src/concurrency/timestamp_ordering_transaction_manager.cpp

```diff
@@ -25,25 +25,48 @@
 namespace peloton {
 namespace concurrency {
 
-// timestamp ordering requires a spinlock field for protecting the atomic access
-// to txn_id field and last_reader_cid field.
+/**
```

Comment:
I'm not seeing how to insert these at the right line numbers. 
Shall we also add Doxygen comments for the following functions in this file? Line numbers on the left. 

218: AcquireOwnership - missing header
271: PerformRead - missing header
535: PerformInsert
695: PerformDelete
781: PerformDelete
1000: AbortTransaction

Reply:
Yes, I didn't comment them since I thought the function names were clear, but we should do it for clarification and consistency. Will add these and fix the one below.